src/service/platform-impl.js

```diff
@@ -79,7 +79,8 @@ export class Platform {
    */
   isIe() {
     return (/MSIE/i.test(this.navigator_.userAgent) ||
```

Comment:
Hoist it out in the same file? And import it in `polyfills/domtokenlist-toggle.js`as in:
```javascript
import {isIe} from '../service/platform-impl'
```

I was worried this might mess up some dependencies but this works for me otherwise.